### PR TITLE
Jweisner/pw 1341/update tumblr public policy repo

### PR DESCRIPTION
**Pull request by JohnathanWeisner**
Updating the terms of service and privacy policies.
New updates include additions to the CCPA section and various Automattic links.

### CODE REVIEW

**Comment by dkfzhf11** [comment]
@[@](@)

**Comment by Muiskraal** [comment]
My new email is smartryk1907@gmail. com I keep my self bizy and love
tumbler sorry for my spelling it is my 3 language. Thanking u Bob Kroukamp

**Review by nalharbi96 — COMMENTED** [review]
G

**Review by nalharbi96 — COMMENTED** [review]
S

**Review by nalharbi96 — COMMENTED** [review]
G

File: terms_of_service/it/terms_of_service.txt [hunk 1]
```diff
@@ -233,7 +233,7 @@ Tumblr fornisce prodotti a livello mondiale e consente a una community globale d
 INVIA UN AVVISO DI VIOLAZIONE 
 
 
-20. DMCA COPYRIGHT POLICY
+20. DMCA Copyright Policy (undefined)
```
Comment (Lrob03) [inline terms_of_service/it/terms_of_service.txt]: terms-of-service.txt

**Pull request merged**